.github/workflows/docker.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Docker
+
+on:
+  push:
+    branches: [ "main" ]
+    tags: [ '*' ]
+  pull_request:
+
+jobs:
+
+  docker:
+    strategy:
+      matrix:
+        version: 
+          - "0.5.1_2"
+
+    uses: famedly/github-workflows/.github/workflows/docker.yml@main
+    with:
+      push: ${{ github.event_name != 'pull_request' }} # Always build, don't publish on pull requests
+      registry_user: famedly-ci
+      registry: docker-oss.nexus.famedly.de
+      image_name: peshming
+      build_args: "VERSION=${{ matrix.version }}"
+      tags: |
+        v-${{ matrix.version }}
```

Comment:
Why the dash after the v? Lgtm otherwise

Reply:
because I'm too tired lol, fixed